lang/syn/Cargo.toml

```diff
@@ -28,5 +29,7 @@ quote = "1"
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 sha2 = "0.10"
+spl-discriminator = { version = "0.1.0", optional = true }
+spl-transfer-hook-interface = { version = "0.3.0", optional = true }
```

Comment:
If I'm not missing anything, these 2 dependencies are only being used for 16 bytes of hardcodable values. If so, that's not very ideal and would be better if we just hardcode it instead.

Reply:
Yeah that's correct. Happy to hard-code here as well.